### PR TITLE
Fix: Asset decimals on select asset screen

### LINKED ISSUE #18414: Wallet: Select Asset Page - use correct decimal formatting
The Select Asset Page is currently rounding decimals to 2 places.

This should be using the same rounding that we currently have on the Wallet Account Page for assets.

cc @OmarBasem

### PR DESCRIPTION
**Pull request by OmarBasem**
fixes: https://github.com/status-im/status-mobile/issues/18414

This PR fixes the asset item decimals on the select asset screen.

**Before**:
Screenshot_20240108_104711_Status

**After**:
Screenshot_20240108_104307_Status

### CODE REVIEW

**Review by J-Son89 — APPROVED** [review]

**Review by smohamedjavid — APPROVED** [review]
🚀

File: src/status_im/contexts/wallet/common/utils.cljs [hunk 1]
```diff
@@ -173,6 +173,12 @@
    constants/optimism-chain-id :optimism
    constants/arbitrum-chain-id :arbitrum})
 
+(defn get-standard-fiat-format
+  [crypto-value currency-symbol fiat-value]
+  (if (string/includes? crypto-value "<")
+    "<$0.01"
+    (prettify-balance currency-symbol fiat-value)))
+
```
Comment (ulisesmac) [inline src/status_im/contexts/wallet/common/utils.cljs:181]: I think we should handle different currncey symbols like for EUR

**Review by ulisesmac — APPROVED** [review]
LGTM

**Review by mmilad75 — APPROVED** [review]

**Review by vkjr — APPROVED** [review]

**Comment by VolodLytvynenko** [comment]
Hi @OmarBasem, thank you for the PR. Apologies for the testing delay. I took this PR for testing and then went on sick leave. Now PR is tested and ready to be merged

**Comment by OmarBasem** [comment]
> Hi @OmarBasem, thank you for the PR. Apologies for the testing delay. I took this PR for testing and then went on sick leave. Now PR is tested and ready to be merged

No worries! Welcome back :)

**Pull request merged**